cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -866,6 +869,18 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
             ));
         }
 
+        Future<ReconciliationState> zkScaleUpStep() {
```

Comment:
I think this is worthy of a Javadoc, to explain the context around how it's used.